packages/components/stories/icon/icons.js

```diff
@@ -54,7 +54,7 @@ export const icons =
         'IcAppstoreWalletUsdLight',
         'IcAppstoreWalletUsdcLight',
         'IcAppstoreWalletUsdtLight',
-        'IcAppstoreWarning'
```

Comment:
This file should not be modified, can u please revert the changes to this file. Please refer to the changes in master

Reply:
it is generated automatically. we should do build after any changes with icons